.xe/specs/blueprint/spec.md

```diff
@@ -12,153 +12,137 @@ dependencies: []
+- **Reusable playbooks over custom prompts**
+  > Structured workflows (playbooks) with defined inputs, outputs, and steps enable consistent execution across platforms. Playbooks are AI-agnostic markdown files, not platform-specific code.
 
 ## Requirements
 
 ### Functional Requirements
 
-- **FR-1**: Blueprint spec MUST document all features identified for Catalyst across all 5 phases
-- **FR-2**: Each feature MUST have a unique kebab-case ID (e.g., `product-context`, `autonomous-orchestration`)
-- **FR-3**: Each feature MUST have a phase assignment matching product.md Product Strategy phases (POC, Mainstream, Innovation, Platform, Enterprise, Scale)
-- **FR-4**: Each feature MUST have dependency tier using phase-relative numbering (e.g., 1.1, 1.2, 2.1) to prevent cascading renumbering when phases evolve
-- **FR-5**: Each feature MUST list dependencies as array of feature IDs it depends on
-- **FR-6**: Each feature MUST have 1-2 sentence scope description defining boundaries
-- **FR-7**: Each feature MUST have complexity estimate (Small, Medium, Large) based on implementation effort
-- **FR-8**: Each feature MUST have priority order number for implementation sequencing
-- **FR-9**: Blueprint spec MUST include feature dependency graph in mermaid format
-- **FR-10**: Feature dependency graph MUST be acyclic (no circular dependencies)
-- **FR-11**: Features in same tier SHOULD be implementable in parallel (no cross-dependencies within tier)
-- **FR-12**: Phase 1 features MUST have detailed scope descriptions; Phase 2-5 MAY have high-level descriptions with "TBD - detail during Phase X planning" notation
-- **FR-13**: Rollout plan MUST track implementation status for all features with checkboxes
-- **FR-14**: Blueprint MUST define core entities that features operate on (context files, templates, playbooks, rollouts)
+**Context Engineering:**
+- **FR-1**: Framework MUST provide centralized context storage in `.xe/` directory
+- **FR-2**: Framework MUST support context files for product vision (product.md), architecture patterns (architecture.md), engineering principles (engineering.md), and development process (process/development.md)
+- **FR-3**: Context files MUST use structured markdown format parseable by AI
+- **FR-4**: Context files MUST be created via project initialization from GitHub issues
+
+**Spec-Driven Development:**
+- **FR-5**: Framework MUST provide playbook engine for executing structured workflows
+- **FR-6**: Playbooks MUST define inputs, outputs, execution steps, checkpoints, and success criteria
+- **FR-7**: Framework MUST support human checkpoints at spec, plan, and tasks milestones
+- **FR-8**: Framework MUST provide templates for specs, plans, tasks, and rollouts
+
+**Feature Implementation:**
+- **FR-9**: Framework MUST support blueprint-based feature breakdown with dependencies, phases, and priorities
+- **FR-10**: Framework MUST enable progressive feature rollout via start-rollout playbook
```

Comment:
Implementation detail. Should we just say that features must be implemented in dependency order?